astroquery/jplhorizons/core.py

```diff
@@ -146,6 +146,7 @@ def ephemerides_async(self, airmass_lessthan=99,
                           refsystem='J2000',
                           closest_apparition=False, no_fragments=False,
                           quantities=conf.eph_quantities,
+                          extra_precision=False,
```

Comment:
please add the new kwarg to the end of the list to keep API backward compatibility for folks using the kwargs as positional args.

Reply:
I knew there was reason to add it at the end - but it's so ugly ... Anyway: done!

Reply:
yes, agreed. Once we drop the ancient python versions it would be possible to make kwarg only arguments, then we can start reordering them to be nicer or more sensible.